src/App.tsx

```diff
@@ -1,61 +1,152 @@
+import original from 'react95/dist/themes/original';
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import {
+  styleReset,
+  Button,
+  Window,
+  WindowHeader,
+  WindowContent,
+} from 'react95';
+
+import { createGlobalStyle, ThemeProvider } from 'styled-components';
+
+/* Pick a theme of your choice */
+
+import { TodoList } from './components/TodoList';
+
+// api
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+// types
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+const GlobalStyles = createGlobalStyle`
+  ${styleReset}
+  @font-face {
+    font-family: 'ms_sans_serif';
+    font-weight: 400;
+    font-style: normal
+  }
+  @font-face {
+    font-family: 'ms_sans_serif';
+    font-weight: bold;
+    font-style: normal
+  }
+  body, input, select, textarea {
+    font-family: 'ms_sans_serif';
+  }
+`;
+
+function getUserById(userId: number): User | null {
+  return usersFromServer.find((user) => user.id === userId)
+  || null;
```

Comment:
```suggestion
  return usersFromServer.find((user) => user.id === userId)
    || null;
```

Reply:
OMG...You kill my project but its great advices. Thank you, i'll deal with it.